shared/img/cursor.svg

```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" xmlns:x="http://www.w3.org/1999/xlink" viewBox="0 0 15 20"><style>use,:target~#orange{display:none}#orange,:target{display:inherit}#orange{fill:#f7a25e}#blue{fill:#00a9dc}</style><defs><g id="cursor"><path d="M3.387 12.783l2.833 5.009.469.829.851-.428 1.979-.995h-.001l.938-.472-.517-.914-2.553-4.513h5.244l-1.966-1.747-9-8-1.664-1.479v17.227-.001l1.8-2.4 1.587-2.116z" fill="#fff"/><path fill-rule="evenodd" d="M3.505 10.96l3.586 6.34 1.979-.995-3.397-6.005h4.327l-9-8v12l2.505-3.34z"/></g></defs><use id="blue" x:href="#cursor"/><use id="orange" x:href="#cursor"/></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:x="http://www.w3.org/1999/xlink" viewBox="0 0 15 20"><style>use,:target~#blue{display:none}#blue,:target{display:inherit}#blue{fill:#00a9dc}#orange{fill:#f7a25e}</style><defs><g id="cursor"><path d="M3.387 12.783l2.833 5.009.469.829.851-.428 1.979-.995h-.001l.938-.472-.517-.914-2.553-4.513h5.244l-1.966-1.747-9-8-1.664-1.479v17.227-.001l1.8-2.4 1.587-2.116z" fill="#fff"/><path fill-rule="evenodd" d="M3.505 10.96l3.586 6.34 1.979-.995-3.397-6.005h4.327l-9-8v12l2.505-3.34z"/></g></defs><use id="blue" x:href="#cursor"/><use id="orange" x:href="#cursor"/></svg>
```

Comment:
`<use id="blue" x:href="#cursor"/>` needs to be the last `<use/>` for the `:target~#blue` rule to correctly hide the blue version when `#orange` is used.